tests/ui/structs/default-field-values-failures.stderr

```diff
@@ -0,0 +1,40 @@
+error: generic parameters may not be used in const operations
+  --> $DIR/default-field-values-failures.rs:22:23
+   |
+LL |     bat: i32 = <Qux<{ C }> as T>::K,
+   |                       ^ cannot perform const operation using `C`
+   |
+   = help: const parameters may only be used as standalone arguments, i.e. `C`
+   = help: add `#![feature(generic_const_exprs)]` to allow generic const expressions
```

Comment:
@Mark-Simulacrum I think is the right approach: it is still gated behind `generic_const_exprs`, but when that feature is enabled it "just works".